frontend/src/components/print/print-nuxt/DownloadNuxtPdfListItem.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-list-item @click="generatePdf">
+    <v-list-item-icon>
+      <v-icon v-if="loading" class="mdi-spin">mdi-loading</v-icon>
+      <v-icon v-else>mdi-nuxt</v-icon>
+    </v-list-item-icon>
+    <v-list-item-title>
+      {{ $tc("components.print.printNuxt.downloadNuxtPdfListItem.label") }}
+    </v-list-item-title>
+  </v-list-item>
+</template>
+
+<script>
+import { saveAs } from 'file-saver'
+import slugify from 'slugify'
+
+import axios from 'axios'
+
+const PRINT_SERVER = window.environment.PRINT_SERVER
+
+export default {
+  name: 'DownloadNuxtPdfListItem',
```

Comment:
Would be good to keep the naming consistent.

If we call this `DownloadNuxtPdfListItem`, then the Button should be called `DownloadNuxtPdfButton`

Reply:
Should I rename it, even though I'll remove the button in next PR targeting the activity.